### DISCUSSION TITLE
Using Docs only mode returns error

### DISCUSSION THREAD

**Discussion by itslevir** [facebook/docusaurus#5401] [Q&A] [answered]
Hi, newbie to docusaurus here,
I'm following the docusaurus docs [here](https://docusaurus.io/docs/docs-introduction#docs-only-mode) and I add `routeBasePath: '/'` to my docusaurus.config.js and it returns this:
```
Error: These field(s) ("routeBasePath",) are not recognized in docusaurus.config.js.
```
For some reason I'm not able to find any fixes or answers to this.

**Comment by Josh-Cena** [comment, marked as answer]
`routeBasePath` is a plugin option. https://docusaurus.io/docs/next/api/plugins/@docusaurus/plugin-content-docs#ex-config-preset Declare it in `presets.docs`.

Multiple people have asked the same question so I guess we should elaborate on this

**Reply by itslevir** [reply]
awesome, thanks!
